src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -158,11 +163,13 @@ object DemuxFastqs {
           batch
             .parWith(pool=pool)
             .map { readRecords => demultiplexer.demultiplex(readRecords: _*) }
+            .map { result => result.maskLowQualityBases(threshold=thresh, qualityEncoding=qualityEncoding) }
             .seq
         }
     }
     else {
       zippedIterator.map { readRecords => demultiplexer.demultiplex(readRecords: _*) }
+        .map { result => result.maskLowQualityBases(threshold=thresh, qualityEncoding=qualityEncoding)}
```

Comment:
Fine to keep them as two maps since there's no parallelism.  Also just a bit of a reformat to see the two `map`s
```suggestion
      zippedIterator
        .map { readRecords => demultiplexer.demultiplex(readRecords: _*) }
        .map { result => result.maskLowQualityBases(threshold=thresh, qualityEncoding=qualityEncoding)}
```